7.redundancy_analysis/notebooks/0.redundancy_analysis.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>Are the a and b terms here supposed to be (x_p - x_mu)^2 and (x_q - x_mu)^2 for the variance? May consider changing this below if this is the case</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I do not think so but let us talk with Greg more on Monday to find out. </p>